src/main/java/com/minecolonies/core/colony/CitizenData.java

```diff
@@ -916,9 +916,33 @@ public void updateEntityIfNecessary()
             }
         }
 
+        //Check if we are traveling, we don't spawn an entity if we are traveling.
+        if (getColony().getTravelingManager().isTravelling(this)) {
```

Comment:
All the formatting is off, I think you need to important the minecolonies formatter =D